runtime/test/storage/pouchdb/pouch-db-key-test.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+/**
+ * Unit tests for PouchDbKey.
+ */
+
+import {PouchDbKey} from '../../../ts-build/storage/pouchdb/pouch-db-key.js';
+
+import 'chai/register-expect';
+
+describe('pouch-db-key', () => {
+  it('fails for non pouchdb: prefixed keys', () => {
+    expect(() => {
+      new PouchDbKey('http://www.google.com/');
+    }).to.throw(Error);
+  });
+
+  it('parses a memory url', () => {
+    let key = 'pouchdb://memory/user/';
+    let pkey = new PouchDbKey(key);
+    expect(pkey.toString()).to.equal(key);
+    expect(pkey.dbLocation).to.equal('memory');
+    expect(pkey.dbName).to.equal('user');
+    expect(pkey.location).to.equal('');
+    expect(pkey.dbCacheKey()).to.equal('memory/user');
```

Comment:
Please use the assert API to be consistent with the rest of the unit tests.

Reply:
done